src/doc_builder/build_doc.py

```diff
@@ -211,7 +212,11 @@ def build_mdx_files(package, doc_folder, output_dir, page_info):
 
         if new_anchors is not None:
             page_name = str(file.with_suffix("").relative_to(doc_folder))
-            anchor_mapping.update({anchor: page_name for anchor in new_anchors})
+            for anchor in new_anchors:
+                if isinstance(anchor, tuple):
+                    anchor_mapping.update({a: f"{page_name}#{anchor[0]}" for a in anchor[1:]})
```

Comment:
When we have several anchors for the same object, the first one is the real anchor name, the other ones, aliases we register.

Reply:
Sounds good!